docs/editors/vim.md

```diff
@@ -375,16 +422,12 @@ This step cleans up resources that are used by the server.
 ## Using an alternative LSP Client
 
 While we recommend using the `coc-metals` extension with `coc.nvim`, Metals will work
-with these alternative LSP clients.
-
-- [`vim-lsc`](https://github.com/natebosch/vim-lsc/): simple installation and
-  low resource usage but limited functionality (no auto-import, cancellation,
-  formatting, folding).
-- [`LanguageClient-neovim`](https://github.com/autozimu/LanguageClient-neovim/):
-  client written in Rust.
-- [`vim-lsp`](https://github.com/prabirshrestha/vim-lsp): simple installation
-  but limited functionality (no auto-import, cancellation and no prompt for
-  build import).
+with these alternative LSP clients. Keep in mind that they have varying levels of LSP support.
```

Comment:
I was working with `vim-lsc` this morning and realized these points weren't really accurate anymore since a lot of functionality has been updated. Same with `vim-lsp`. I think it might be best to just list what they are, and not really mention features for each one, but rather put a general note to keep in mind that there are varying levels of support.